lib/LaTeXML/Core/Definition/Expandable.pm

```diff
@@ -96,7 +96,7 @@ sub invoke {
           && (($r eq 'LaTeXML::Core::Token') || ($r eq 'LaTeXML::Core::Tokens'))
         ? $_ : Tokens(Revert($_))); } @args;
     if ($tracing || $LaTeXML::DEBUG{tracing}) {    # More involved...
-      Debug($self->tracingCSName . ' -> ' . tracetoString($expansion));
+      Debug($self->tracingCSName . ' ->' . tracetoString($expansion));
```

Comment:
These space removals are not typos - TeX doesn't have a space after these two arrows, so it's easier to diff without that space added in latexml.